scripts/docker/cpp-build-env.dockerfile

```diff
@@ -0,0 +1,30 @@
+# build:          docker build -t test_build_20 .
+# run:            docker run -it --rm test_build_20:latest
+# versioning:     docker tag cpp-build-env ethereum/cpp-build-env:v0.8.29.2017
+# upload:         docker push tcsiwula/cpp-build-env
+
+# get latest stable debian release image -- see https://hub.docker.com/_/debian/
+from debian:latest
+
+# maintainer email
+MAINTAINER Tim Siwula <tcsiwula@gmail.com>
```

Comment:
`MAINTAINER` is deprecated. Use `LABEL` (read the docs). And point it to C++ Ethereum developers team (no email at the moment, but you can link cpp-ethereum repo.

Reply:
^ good catch